print/ghostscript/Portfile

```diff
@@ -141,13 +158,31 @@ post-destroot {
             COPYING ChangeLog README README.tweaks TODO \
             ${destroot}${prefix}/share/doc/${name}/fonts
 
+    if {[variant_isset ghostpdl]} {
+        # Install PCL base fonts into urwfonts directory
+        xinstall -m 755 -d ${destroot}${prefix}/share/${name}/urwfonts
+        xinstall -m 644 {*}[glob -directory ${worksrcpath}/pcl/urwfonts *.ttf] \
+            ${destroot}${prefix}/share/${name}/urwfonts
+    }
+
     # install missing header
     xinstall -m 0644 ${worksrcpath}/base/gserrors.h \
             ${destroot}${prefix}/include/ghostscript/
 }
 
 post-activate {
     system "${prefix}/bin/fc-cache -v ${prefix}/share/fonts/Type1/gsfonts"
+
+    if {[variant_isset ghostpdl]} {
+        ui_msg "############################################################"
+        ui_msg "The PCL interpreter requires access to the base 80 font set"
+        ui_msg "for proper operation. To use the default urw fonts, add the"
+        ui_msg "following to your shell profile:"
+        ui_msg ""
+        ui_msg "    export PCLFONTSOURCE=${prefix}/share/${name}/urwfonts/"
+        ui_msg ""
+        ui_msg "############################################################"
+    }
```

Comment:
Change the `ui_msg` lines into a `notes` block. Also remove the "############################################################" lines entirely.